Automattic/wp-calypso
# Jetpack Connect: Remove Connect Header component

This PR aims to remove the `ConnectHeader` component - this was suggested by @johnHackworth in https://github.com/Automattic/wp-calypso/pull/8326#issuecomment-250734322. That component appears to serve no specific purpose and can be removed without any consequences. If we merge this PR, we should close #8326.

To test:

* Checkout this branch
* Verify there are no changes in the JPC flows.

/cc @johnHackworth @roccotripaldi
